### DISCUSSION TITLE
Help with rooting my device

### DISCUSSION THREAD

**Discussion by RaptaG** [openandroidinstaller-dev/openandroidinstaller#334] [Q&A] [answered]
I made the mistake to not backup my photos since I though that MI Unlock wouldn't unlock my device instantly and instead give me the 7 day delay as usual. Either way, I found [a promising solution on retrieving them](https://xdaforums.com/t/lost-my-childs-sweet-16-pics-flashing-a-rom-on-moto-g.2880897/post-55541464).

But I have a problem. My device is not rooted and I don't know how to root it. Can it be done through OAI without installing a ROM (I'll do that after I restore the photos). What should I be informed of?

**Comment by tsterbak** [comment, marked as answer]
Hey, I'm sorry to hear, that you lost your data. :/ I'm not sure if there is a way to get them back if a factory reset is performed.

Unfortunately, I have only very limited experience with rooting devices since I usually don't do it. Also, the OAI won't help with rooting your device at all. It's only meant to bring a new ROM on the device.

I would recommend looking for tutorials on xda or reddit on how to root your specific device. Good luck! :)

**Reply by tsterbak** [reply]
Also, be careful when using `dd` somewhere. If you type something wrong, it might also wipe your PC.
